test/consul.js

```diff
@@ -346,57 +346,6 @@ describe('Consul', () => {
     });
   });
 
-  it('resolves multiple tags as separate services', () => {
```

Comment:
Is this test case (agents with different IP addresses and tags registering the same service should resolve all the addresses to the service) covered elsewhere? If not, we might want to fix it so it passes instead of removing it.